### ISSUE TITLE
Response body of examples losts during import

### ISSUE THREAD

**Issue by galbarm** [a85/POSTMan-Chrome-Extension#5029]
Reproduces with version 6.2.4 for windows.

When importing a collection with a request that contains an example with a request body, the request body is not imported with it.

For example, look at this request and the body of the example:
![capture](https://user-images.githubusercontent.com/4610728/44141200-16816da4-a085-11e8-8e07-61872cadb364.JPG)

As you can see it has concrete values.
Following is the json output of exporting the collection as version 2.1

```
{
	"info": {
		"_postman_id": "39f3e53f-8283-440b-9257-3171d2a76862",
		"name": "ExamplesImportBugDemonstration",
		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
	},
	"item": [
		{
			"name": "Job Url Copy",
			"event": [
				{
					"listen": "prerequest",
					"script": {
						"id": "54eba049-6f9d-42ea-9030-04e4866b0c2c",
						"type": "text/javascript",
						"exec": [
							""
						]
					}
				},
				{
					"listen": "test",
					"script": {
						"id": "8ec08532-4457-4192-b0cd-7a0500c0548b",
						"type": "text/javascript",
						"exec": [
							""
						]
					}
				}
			],
			"request": {
				"method": "POST",
				"header": [
					{
						"key": "",
						"value": ""
					},
					{
						"key": "Content-Type",
						"value": "application/json"
					}
				],
				"body": {
					"mode": "raw",
					"raw": "{\r\n   \"name\":\"{{jobName}}\",\r\n   \"project\" : \"{{projectId}}\",\r\n   \"input\":{\r\n      \"type\":\"{{jobInputType}}\",\r\n      \"info\":{\r\n        {{jobInputInfo}}\r\n      }\r\n   },\r\n   \"output\":{\r\n      \"type\":\"{{jobOutputType}}\",\r\n      \"info\":{\r\n        {{jobOutputInfo}}\r\n      }\r\n   },\r\n   \"configuration\":{\r\n      \"processors\":[\r\n        {\r\n          \"name\": \"{{processorName}}\",\r\n          \"info\": {\r\n            {{processorInfo}}\r\n          }\r\n        }\r\n      ]\r\n   }\r\n}"
				},
				"url": {
					"raw": "{{protocol}}://{{host}}/api/prism/jobs/url",
					"protocol": "{{protocol}}",
					"host": [
						"{{host}}"
					],
					"path": [
						"api",
						"prism",
						"jobs",
						"url"
					]
				},
				"description": "Request a job url to be generated.\nconnecting with the given url is then used to start the job.\nAccording to info in the body, the generated url will either be http or ws.\nThe jobId is also returned here.\nproject id can be provided - not mandatory."
			},
			"response": [
				{
					"name": "StorageService, NightKing Processor",
					"originalRequest": {
						"method": "POST",
						"header": [
							{
								"key": "Authorization",
								"value": "Bearer {{accessToken}}"
							},
							{
								"key": "Content-Type",
								"value": "application/json"
							},
							{
								"key": "X-Account-ID",
								"value": "{{accountToken}}"
							}
						],
						"body": {
							"mode": "raw",
							"raw": "{\r\n   \"name\":\"MyJob\",\r\n   \"input\":{\r\n      \"type\":\"StorageService\",\r\n      \"info\":{\r\n         \"volume\":\"4fe4d4aa41f4\",\r\n         \"files\":[\r\n            \"0ed5f89f718b\"\r\n         ]\r\n      }\r\n   },\r\n   \"output\":{\r\n      \"type\":\"StorageService\",\r\n      \"info\":{\r\n         \"volume\":\"4fe4d4aa41f4\"\r\n      }\r\n   },\r\n   \"configuration\":{\r\n      \"processors\":[\r\n         {\r\n            \"name\":\"ProcessorSimulator.jar\",\r\n            \"info\":{\r\n               \"type\":\"king\"\r\n            }\r\n         }\r\n      ]\r\n   }\r\n}"
						},
						"url": {
							"raw": "{{protocol}}://{{host}}/api/prism/jobs/url",
							"protocol": "{{protocol}}",
							"host": [
								"{{host}}"
							],
							"path": [
								"api",
								"prism",
								"jobs",
								"url"
							]
						}
					},
					"status": "OK",
					"code": 200,
					"_postman_previewlanguage": "json",
					"header": [
						{
							"key": "Content-Encoding",
							"value": "gzip",
							"name": "Content-Encoding",
							"description": "The type of encoding used on the data."
						},
						{
							"key": "Content-Length",
							"value": "993",
							"name": "Content-Length",
							"description": "The length of the response body in octets (8-bit bytes)"
						},
						{
							"key": "Content-Type",
							"value": "application/json",
							"name": "Content-Type",
							"description": "The mime type of this content"
						},
						{
							"key": "Date",
							"value": "Sun, 10 Jun 2018 09:19:57 GMT",
							"name": "Date",
							"description": "The date and time that the message was sent"
						},
						{
							"key": "Vary",
							"value": "Origin,Accept-Encoding",
							"name": "Vary",
							"description": "Tells downstream proxies how to match future request headers to decide whether the cached response can be used rather than requesting a fresh one from the origin server."
						},
						{
							"key": "X-Request-ID",
							"value": "361ce8bd-c3ef-4529-86b6-899e349b0f14",
							"name": "X-Request-ID",
							"description": "Custom header"
						}
					],
					"cookie": [],
					"body": "{\n    \"url\": \"https://localhost:9012/api/prism/jobs/strict/eyJraWQiOiJmbGlyIiwiYWxnIjoiUlMyNTYifQ.eyJqb2JJZCI6ImxhbWJkYTpwcmlzbTpkZXY6ODIwMDRiYWEtZjc4OC00ZGJlLWJiYTYtZTZiNjY4Yzk2YzRiOmpvYjozNjFjZThiZC1jM2VmLTQ1MjktODZiNi04OTllMzQ5YjBmMTQiLCJjbGllbnRDb250ZXh0Ijp7ImNsaWVudCI6IjAwMDAwMDAwLTAwMDAtMDAwMC0wMDAwLTAwMDAwMDAwMDAwMSIsInNlc3Npb25JZCI6IjUwNDc0MWI0LTdiOGUtNDkwOS04Y2VmLTUwY2RlMjMzYmE5OSIsInJvbGVzIjpbIkFkbWluIl19LCJhY2NvdW50Q29udGV4dCI6eyJhY2NvdW50SWQiOiI4MjAwNGJhYS1mNzg4LTRkYmUtYmJhNi1lNmI2NjhjOTZjNGIiLCJhY2NvdW50VHlwZSI6ImZsaXIiLCJ1c2VyIjoiMDAwMDAwMDAtMDAwMC0wMDAwLTAwMDAtMDAwMDAwMDAwMDAxIiwic2Vzc2lvbklkIjoiNTA0NzQxYjQtN2I4ZS00OTA5LThjZWYtNTBjZGUyMzNiYTk5Iiwicm9sZXMiOltdfSwiZmxvd0lkIjoiMzYxY2U4YmQtYzNlZi00NTI5LTg2YjYtODk5ZTM0OWIwZjE0IiwiaXNzIjoiTGFtYmRhU2VydmljZXMiLCJhdWQiOiJMYW1iZGFTZXJ2aWNlcyIsImp0aSI6ImREbzZnV1BwQ05vSGpRSmdZMVZMUHciLCJpYXQiOjE1Mjg2MjIzOTcsImV4cCI6MTUyODYyNTk5N30.gnv_-xgAyT_BeTz1JIb1TD5DqKeMMDUrQhp4WuLfeoswZX7roYvE0Y43XyIGWfp8DSw1Kq2YZxKcmKsWN7IuDr_n_IuEqU-dmGyValm6NIYencUD2-9Us-uANH3-7pWaZKmamCK9y8uPf0bQ4PLsjq7YzBz9gcAtKZ7eCQl2auP2raYE6vCLpxKNHEXw0DQgsQb4afm7IKvOZe7-rWaX-GoLXhSeJnLrRP7rEO8fRZlM81HplfhQaOSHo4CDG5dse3xXL8uvohBzjUV4SveVipguJAorkvBDOgG6nwZAAsIrdjDrE_8nNJxqslju-AxMrp-YPlxc4DRkXHLXw4cUDw\",\n    \"jobId\": \"lambda:prism:dev:82004baa-f788-4dbe-bba6-e6b668c96c4b:job:361ce8bd-c3ef-4529-86b6-899e349b0f14\"\n}"
				}
			]
		}
	]
}
```

If you look at the example in the json, you can see that it still contains the concrete values of the request body.
But if you take this json and import it, The example in the imported collection loses the values and looks like this:
![capture](https://user-images.githubusercontent.com/4610728/44141451-db06e5c8-a085-11e8-9dfb-b94481225403.JPG)

It is showing the request body of the request itself.

**Comment by galbarm** [comment]
Might be related to https://github.com/postmanlabs/postman-app-support/issues/3723

**Comment by galbarm** [comment]
Also, this seems to be new.
My colleague downgraded to 6.1 because of this, and says it doesn't reproduce there.

**Comment by numaanashraf** [comment]
@galbarm Thanks for reporting this. This is a confirmed regression and we are working on fixing this ASAP.

**Comment by harryi3t** [comment]
@galbarm Thank you for reporting this issue. We have found the root cause and have fixed it in an internal build. We are testing for stability and this fix would be available in the next stable release.

Here's a short post-mortem about the bug

#### Q. What was the bug?
In Postman version 6.2.x while importing a collection with examples, the example's request (url, header, body, scripts, etc) was being dropped and was being replaced with the actual request.

#### Q. Why did it happen?
In 6.2.1 we did a minor refactoring to improve the import flow. While doing this, we missed out a case for this particular data which needed special handling.

#### Q. How are we going to prevent this from happening in the future?
We have handled this case correctly now and have added automated tests for this case so that it would not regress in the future.

**Comment by harryi3t** [comment]
@galbarm This has been fixed in the latest version of Postman 6.2.5

**Comment by galbarm** [comment]
This happens again in 6.4.4.
Take the json from above, import it, the request body of the example is empty.
